src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -143,6 +143,11 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
      */
     protected function extractProperties(Request $request/*, string $resourceClass*/): array
     {
-        return $request->query->get($this->orderParameterName, []);
+        $properties = $request->query->get($this->orderParameterName, []);
+        if (!is_array($properties)) {
+            return [];
```

Comment:
Plus the built-in API documentation is pretty clear on how to send order parameters.